sklearn/tests/test_multiclass.py

```diff
@@ -426,6 +443,21 @@ def test_ovo_fit_predict():
     assert_equal(len(ovo.estimators_), n_classes * (n_classes - 1) / 2)
 
 
+def test_ovo_partial_fit_predict():
+    X, y = shuffle(iris.data, iris.target)
+    ovo1 = OneVsOneClassifier(MultinomialNB())
+    ovo1.partial_fit(X[:100], y[:100], np.unique(y))
+    ovo1.partial_fit(X[100:], y[100:])
+    pred1 = ovo1.predict(X)
+
+    ovo2 = OneVsOneClassifier(MultinomialNB())
+    ovo2.fit(X, y)
+    pred2 = ovo2.predict(X)
+    assert_equal(len(ovo1.estimators_), n_classes * (n_classes - 1) / 2)
+    assert_greater(np.mean(y == pred1), 0.65)
```

Comment:
They are basically doing the same thing, i.e finding mean by comparing `real_y` and `predicted_y`, so I don't think it is necessary to do that. Infact I believe using `assert_greater(np.mean(y == pred1), 0.65)` is more explicit than `ovo1.score(X,y)`.